### ISSUE TITLE
Tutorial: Spin Hall

### ISSUE THREAD

**Issue by jaemolihm** [wannier-berri/WannierBerri-tutorial#3]
https://github.com/wannier-berri/Wannier2022/blob/main/advanced/spin_hall/tutorial_spin_hall.ipynb

I added a draft for the notebook for the spin Hall tutorial.

@manxkim

**Comment by jaemolihm** [comment]
@manxkim Please do not delete files. They are necessary to reproduce calculations.

**Comment by manxkim** [comment]
@jaemolihm It was to make the structure of directories similar to basic. Does moving the output files into the separated directories affect calculations? Are the files other than .chk, .eig, .mmn, .spn, .sHu, .sIu needed for wberri.System_w90?

**Comment by manxkim** [comment]
@jaemolihm One more: how about not adding kBT smearing part? The parameters for (temperature and also frequency) smearing are already inside the dynamic SHC part.

**Comment by jaemolihm** [comment]
> Does moving the output files into the separated directories affect calculations?

No, but the files were deleted. (Maybe you did not `git add`?) Anyway, I think we can just keep them as is now.

> Are the files other than .chk, .eig, .mmn, .spn, .sHu, .sIu needed for wberri.System_w90?

No, but they are useful for learning and for reproducibility.

> One more: how about not adding kBT smearing part? The parameters for (temperature and also frequency) smearing are already inside the dynamic SHC part.

Sounds alright.

**Comment by manxkim** [comment]
> One more: how about not adding kBT smearing part? The parameters for (temperature and also frequency) smearing are already inside the dynamic SHC part.

Then, let's add a comment to the code
```
# In case you want smeared data, use instead
# shc_ryoo = result.results["SHC_ryoo"].smoothers[0, :, 0, 1, 2]
# shc_qiao = result.results["SHC_qiao"].smoothers[0, :, 0, 1, 2]
```
